TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
+  
+  private let markedSpaces: [Int: PlayerMark]
+  private let size = 3
+
+  public init() {
+    markedSpaces = [:]
+  }
+
+  public init(markedSpaces: [Int : PlayerMark]) {
+    self.markedSpaces = markedSpaces
+  }
+
+  public func markSpace(id: Int, mark: PlayerMark) -> Board {
```

Comment:
A space ID. I'll rename to spaceId. Longer term I'm considering ways of making referencing spaces more expressive/flexible than by using integer ids.